images/encryption/encryption.go

```diff
@@ -230,7 +209,7 @@ func ingestReader(ctx context.Context, cs content.Ingester, ref string, r io.Rea
 		return "", 0, errors.Wrap(err, "failed to get state")
 	}
 
-	if err := cw.Commit(ctx, st.Offset, ""); err != nil {
+	if err := cw.Commit(ctx, st.Offset, cw.Digest()); err != nil {
```

Comment:
don't call `cw.Digest()` here, it should be called after `Commit`

Reply:
Ok